tests/integration_tests/test_event_loop_integration.py

```diff
@@ -24,15 +37,33 @@ def test_event_loop_integration_with_one_iteration(
 def test_event_loop_integration_with_two_iteration(
         setup_database, pyexasol_connection, upload_language_container):
 
+    # get audit logs before executing event loop
+    latest_stmt = datetime.utcnow()
```

Comment:
Timezones can be different between database and test. Instead, run a unique query with a long uuid in it. Query the audit log for this unique query and use its start time.